src/components/DefaultRelatedContent.js

```diff
@@ -12,37 +20,37 @@ const DefaultRelatedContent = () => {
         >
           <Icon focusable={false} name="fe-link-2" size="1rem" />
         </Link>
-        For more help
+        {t('defaultRelatedContent.title')}
       </h2>
-      <p>
-        If you need more help, check out these support and learning resources:
-      </p>
+      <p>{t('defaultRelatedContent.intro')}</p>
       <ul>
-        <li>
-          Browse the{' '}
-          <ExternalLink href="https://discuss.newrelic.com">
-            Explorers Hub
-          </ExternalLink>{' '}
-          to get help from the community and join in discussions.
-        </li>
-        <li>
-          Find{' '}
-          <a href="/docs/using-new-relic/welcome-new-relic/get-started/find-help-use-support-portal">
-            answers on our sites and learn how to use our support portal
-          </a>
-          .
-        </li>
-        <li>
-          Run{' '}
-          <a href="/docs/using-new-relic/cross-product-functions/troubleshooting/new-relic-diagnostics">
-            New Relic Diagnostics
-          </a>
-          , our troubleshooting tool for Linux, Windows, and macOS.
-        </li>
-        <li>
-          Review New Relic's <a href="/docs/security">data security</a> and{' '}
-          <a href="/docs/licenses">licenses</a> documentation.
-        </li>
+        <Trans i18nKey="defaultRelatedContent.list">
```

Comment:
```suggestion
        <Trans i18nKey="defaultRelatedContent.list" parent="ul">
```

You might consider making the `Trans` the `ul` instead of nesting it, since `ul` doesn't take any props.

Reply:
ah gotcha. makes sense to me and makes things slightly cleaner